test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/FilterAndOrderByFunctionalTests.cs

```diff
@@ -1142,6 +1142,22 @@ public void AggregatedPropertyTreatedAsOpenPropertyInOrderBy()
             orderByClause.Expression.ShouldBeSingleValueOpenPropertyAccessQueryNode("Total");
         }
 
+        [Fact]
+        public void ComputedPropertyTreatedAsOpenPropertyInOrderBy()
+        {
+            var odataQueryOptionParser = new ODataQueryOptionParser(HardCodedTestModel.TestModel,
+                HardCodedTestModel.GetPersonType(), HardCodedTestModel.GetPeopleSet(),
+                new Dictionary<string, string>()
+                {
+                    {"$orderby", "DoubleTotal asc"},
+                    {"$apply", "aggregate(FavoriteNumber with sum as Total)/compute(Total mul 2 as DoubleTotal)"}

```

Comment:
>DoubleTotal [](start = 99, length = 11)

What would happen if you renamed it to `Total`?

Reply:
Good question and we have the same issue with aggregate. Not going to fix it as part of this PR